hedgedocs/docs/nn/common/chunk-format/nodenamelist.md

```diff
@@ -0,0 +1,59 @@
+---
+description: Sega NN NODENAMELIST Data Chunk Format Specification
+---
+
+# NODENAMELIST
+
+## NNS_NODENAMELIST
+
+!!! important
+    This is the "main" (root) struct for all NN node (bone) name list chunks.
+
+```c
+struct NNS_NODENAMELIST
+{
+    /* The sorting type */
+    NNE_NODENAME_SORTTYPE SortType;
+
+    /* The size of [pNodeNameList] array */
+    uint32_t nNode;
+
+    NNS_NODENAME* pNodeNameList;
+};
+```
+
+!!! todo
+    Find the macros used in the text-form file format
+
+## NNS_NODENAME
+
+The actual node (bone) name
+
+```c
+struct NNS_NODENAME
+{
+    /* The node index */
+    int iNode;
+
+    /* The node name */
+    char* Name;
+};
+```
+
+## NNE_NODENAME_SORTTYPE
+
+This enum indicates the sorting type used in the node name list.
+
+```c
+enum NNE_NODENAME_SORTTYPE
```

Comment:
Should probably specify what base type this uses. I forget if this is a uint32_t or a uint8_t that's followed by padding? Only matters for big-endian files.

If you're not sure, don't worry about it, we can just assume it's a uint32_t.